publications/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.conf import settings
+from .fields import PrivateFileField
+
+
+def upload_to(instance, filename):
+    return f'pdfs/{filename}'
+
+
+class PDFFile(models.Model):
+    title = models.CharField(_('Title'), max_length=250)
+    file = PrivateFileField(upload_to=upload_to)
```

Comment:
We probably don't want to store these files in private storage as publications are intended to be publicly viewable.